runtime/outer-PEC.js

```diff
@@ -114,9 +114,11 @@ class OuterPEC extends PEC {
     particle.spec.exposes.forEach(
       expose => exposeMap.set(expose.name, views.get(expose.view)));
     
-    var particleSpec = {particle, views, renderMap, exposeMap }
+    var particleSpec = {particle, views, renderMap, exposeMap };
 
-    this._apiPort.InstantiateParticle(particleSpec, { particleName: particle.name, views })
+    this._apiPort.InstantiateParticle(particleSpec, { particleName: particle.name, views });
+
+    return particleSpec;
```

Comment:
I'm not sure this is the right thing to return here, but it clearly is expected to return something:
https://github.com/PolymerLabs/arcs/blob/596fc4ea726e98a1f1e019489aa5f36dfa566349/runtime/arc.js#L161